test/controllers/SurveyControllerSpec.scala

```diff
@@ -89,4 +89,55 @@ class SurveyControllerSpec extends ControllerBaseSpec {
 
   }
 
+  "posting invalid survey data" should {
+    "return a bad request" in new SurveyControllerTest {
+
+      val testModel: ExitSurveyAuditModel = ExitSurveyAuditModel(SurveyJourneyModel(Some("yes"),
```

Comment:
The vals _testModel_, _expectedData_ and the _sendEvent_ mocked function are the same in these three unit tests. Could you put these in the trait or elsewhere outside of the tests?